packages/client/src/links/core.ts

```diff
@@ -46,13 +46,25 @@ export interface HTTPLinkOptions {
 export type HttpLinkOptions = HTTPLinkOptions;
 
 export type HTTPHeaders = Record<string, string | string[] | undefined>;
+
+/**
+ * The default `fetch` implementation has an overloaded signature. By convention this library
+ * only uses the overload taking a string and options object.
+ */
+export type TRPCFetch = (
+  url: string,
+  options?: RequestInit,
+) => Promise<Response>;
+
 export type LinkRuntimeOptions = Readonly<{
   transformer: DataTransformer;
   headers: () => HTTPHeaders | Promise<HTTPHeaders>;
-  fetch: typeof fetch;
+  fetch: TRPCFetch;
   AbortController?: typeof AbortController;
 }>;
 
+export { getFetch } from '../internals/fetchHelpers';
```

Comment:
Move `getFetch()` out of the `internals/` folder

Reply:
Where do you think it should go? `src/links/` feels wrong, should I make a `getFetch.ts` in `src/`?

Reply:
Pick one, I don't mind, it won't change the interface